@here/harp-mapview/lib/Utils.ts

```diff
@@ -665,13 +638,18 @@ export namespace MapViewUtils {
     }
     /**
      * @internal
-     * @deprecated This method will be moved to MapView.
```

Comment:
Who put this here, and do you know why?

Does it still make sense to keep this deprecated so that it will eventually be moved to MapView.. or not?

Reply:
Nino deprecated several functions in MapView's Utils because there were not usages outside MapView at that time, here:
https://github.com/heremaps/harp.gl/pull/1365

But now getTargetAndDistance() is used within Utils.